contributors/design-proposals/kubelet-authorizer.md

```diff
@@ -146,29 +142,39 @@ for compatibility with deployment methods that bind other users or groups to tha
 
 ### Kubelets outside the `system:nodes` group
 
-Kubelets outside the `system:nodes` group would not be authorized by the node authorizer,
+Kubelets outside the `system:nodes` group would not be authorized by the `Node` authorization mode,
 and would need to continue to be authorized via whatever mechanism currently authorizes them.
 The node admission plugin would not restrict requests from these kubelets.
 
 ### Kubelets with undifferentiated usernames
 
 In some deployments, kubelets have credentials that place them in the `system:nodes` group,
 but do not identify the particular node they are associated with.
-Those kubelets would be broadly authorized by the node authorizer,
-but would not have secret and configmap requests restricted.
-The node admission plugin would not restrict requests from these kubelets.
+
+These kubelets would not be authorized by the `Node` authorization mode,
+and would need to continue to be authorized via whatever mechanism currently authorizes them.
+
+The node admission plugin *would* restrict requests from these kubelets.
+That plugin should not be used in a deployment with kubelets that do not identify themselves.
```

Comment:
after discussing the roll-out plan with @mikedanese, @cjcullen, and @deads2k, I'm ok with changing the node identifier to require both a group and username format match in order for the authorizer and admission plugin to pay attention to the request. See https://github.com/kubernetes/kubernetes/pull/46796/

Updated the proposal to match.